docs/overview/multiparty/broadcast.md

```diff
@@ -23,7 +18,7 @@ often that needs to include certain reference data that is available
 to all parties in the network. The data needs to be "broadcast" to all
 members, and also need to be available to new members that join the network
 
-![Multi-party Systems](../images/multiparty_system.png "Multi-Party System")
+![Multi-party Systems](../../images/multiparty_system1.png "Multi-Party System")
 
 ## Blockchain backed broadcast
 
```

Comment:
Line https://github.com/hyperledger/firefly/pull/1055/files#diff-73fbeb2c7d5baa4eca06e28e0b0d069f385d87a96a8d77958f6a05ffd84266a6R33 looks like it should be a real link for `global sequencing` though I can't locate a section in the docs of that name.

Reply:
Similarly for https://github.com/hyperledger/firefly/pull/1055/files#diff-73fbeb2c7d5baa4eca06e28e0b0d069f385d87a96a8d77958f6a05ffd84266a6L67-R66

Reply:
Found and fixed the link - it's `Multiparty Event Sequencing`

Reply:
Resolved the other three links too 👍